test/unit/product.js

```diff
@@ -356,7 +356,7 @@ describe('Product class', () => {
 
       it('calls fetchProduct with product id', () => {
```

Comment:
I'll stick with storefrontId for this test, and make a separate test for the normalization.